scripts/build.py

```diff
@@ -146,6 +145,7 @@ def get_cache_slug():
     ASSETS_OUT_DIR = os.path.join(BUILD_DIR, 'assets', '')
     copy_files_source_to_target(ASSETS_SRC_DIR, ASSETS_OUT_DIR)
 
+    process_third_party_libs()
```

Comment:
No, we do not need /generated/prod for anything. Okay I'll work on that as part of this PR and update the issue.